src/gluonts/mx/block/decoder.py

```diff
@@ -30,9 +30,7 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
     # noinspection PyMethodOverriding
-    def hybrid_forward(
-        self, F, dynamic_input: Tensor, static_input: Tensor
-    ) -> None:
+    def hybrid_forward(self, F, dynamic_input, static_input):
```

Comment:
Why remove the type annotations?

Reply:
Truth is, there appear to be no way of making mypy happy here

Reply:
addressed, there was actually some inconsistencies of ours in all this subclassing